src/smc-webapp/client_browser.coffee

```diff
@@ -43,10 +44,12 @@ misc = require('smc-util/misc')
 idle_notification_html = ->
     {redux}   = require('./app-framework')
     customize = redux.getStore('customize')
+    site_name = customize.get('site_name') or SITE_NAME
+    logo_url = customize.get('logo_square') or APP_LOGO_WHITE
     """
     <div>
-    <img src="#{APP_LOGO_WHITE}">
-    <h1>Collaborative Calculation</h1>
```

Comment:
In theme we have:

```
exports.SITE_NAME = "CoCalc";
exports.APP_TAGLINE = "Collaborative Calculation and Data Science";
```

So you're replacing "Collaborative Calculation" by CoCalc, and the result is going to be a big logo that says "CoCalc" with the words "CoCalc" right below it.  So... shouldn't the new heading be the tagline instead?

Reply:
ah, I see, the white logo also has the name. this must be done like on the about page.

Reply:
I don't know what you mean.  Incidentally, this file `src/smc-webapp/client_browser.coffee` is about to disappear as part of my client.coffee rewrite...

Reply:
So I'm going to merge this PR first as is, to avoid any merge conflict problems.